lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandler.java

```diff
@@ -320,14 +324,15 @@ private IdentityVerificationService createIdentityVerificationService(
                 new FormDataValidator(), null, serviceFactory.getEventProbe());
     }
 
-    private ThirdPartyAPIService selectThirdPartyAPIService() {
+    private ThirdPartyAPIService selectThirdPartyAPIService(
+            boolean dvaDirectEnabled, String documentCheckingRouteHeader) {
         // TODO Change to a switch for each api and read feature flag + header
-        if (configurationService.getUseLegacy()) {
-            // Legacy DCS
-            return thirdPartyAPIServiceFactory.getDcsThirdPartyAPIService();
-        } else {
+        if (dvaDirectEnabled && "dvad".equalsIgnoreCase(documentCheckingRouteHeader)) {
```

Comment:
To avoid confusion can we make this "dvad" - "dvadirect"
Unfortunately dvad (DVA direct) is the same acronym as hmpo

Reply:
Corrected.